Formula/brev-cli.rb

```diff
@@ -1,18 +1,21 @@
 class BrevCli < Formula
   desc "CLI tool for managing workspaces provided by brev.dev"
   homepage "https://docs.brev.dev"
-
   url "https://github.com/brevdev/brev-cli/archive/refs/tags/v0.6.12.tar.gz"
   license "MIT"
-
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-o", "brev",  "-ldflags", "\"-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v.0.6.12\""
+    system "go",
+           "build",
+           "-o",
+           "brev",
+           "-ldflags",
+           "\"-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v.0.6.12\""
     bin.install "brev"
   end
 
   test do
-    system "#{bin}/brev", "--version"
+    system "#{bin}/brev", " --version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I can make a healthcheck to our backend here instead, most of the commands of the application are for an authed user